src/packets.toit

```diff
@@ -5,12 +5,38 @@
 import bytes
 import binary
 import reader
+import .client show Client  // For toitdoc reference.
 
 import .topic_filter
 
 interface PacketIDAck:
   packet_id -> int
 
+class LastWillConfig:
+  retain/bool
```

Comment:
Does our style guide suggest having a space before / for these types?

Reply:
yes, but the whole package was written without spaces...